debian/install

```diff
@@ -2,6 +2,5 @@ bin/ubuntu-core-initramfs usr/bin
 postinst.d etc/kernel/
 snakeoil/* usr/lib/ubuntu-core-initramfs/snakeoil/
 debian/tmp/* usr/lib/ubuntu-core-initramfs/main
-debian/tmp/usr/lib/systemd/boot/efi/linux*.efi.stub usr/lib/ubuntu-core-initramfs/efi/
-debian/sbat.txt usr/lib/ubuntu-core-initramfs/efi/
```

Comment:
note that our copy of sbat.txt is still needed to correctly assemble kernel.efi that contains sbat section.